src/compiler/checker.ts

```diff
@@ -985,6 +985,7 @@ import {
     setNodeFlags,
     setOriginalNode,
     setParent,
+    setSourceMapRange,

```

Comment:
I feel like it's suspicious that the checker did not already import this but presumably gets things right for other nodes...

Reply:
Heh, ye - definitely. I was quite surprised that this was the first time this had to be imported here.

Usually, this gets called by *transforms* but this case is different than that use case. There might also be other issues like this one. In general, declaration maps bugs are less likely to be reported than typechecking bugs etc so 🤷